lib/active_merchant/billing/gateways/rapyd.rb

```diff
@@ -188,6 +199,10 @@ def add_payment_fields(post, options)
         post[:payment][:statement_descriptor] = options[:statement_descriptor] if options[:statement_descriptor]
       end
 
+      def add_customer_object(post, payment, options)
```

Comment:
Nit One: seems like `add_customer_string` and not `add_customer_object`. Or simply `add_name`.

Nit Two: `options` is not used in this method.

Reply:
I named this method to match the [Rapyd Customer Object](https://docs.rapyd.net/build-with-rapyd/reference/customer-object). Even though we are only utilizing it for the name, for now, it felt appropriate to "plan ahead" in case we want to pass more fields.

That should explain why I had an `options` argument here, but I removed it since it is not being used.